docs/instances.md

```diff
@@ -475,7 +475,7 @@ Note, this log location is different from many other Linux conventions, such as
 Within a BOSH instance you can try watching all the logs using `tail`:
 
 ```
-$ tail -f /var/vcap/sys/log/{*.log,*/*.log}
```

Comment:
Sadly this can sometimes be true - that logs aren't made more easily available to less privileged users.